tests/enchant_providers/Dictionary/dictionary_suggest.cpp

```diff
@@ -167,6 +167,11 @@ TEST_FIXTURE(DictionarySuggest_TestFixture,
           i != suggestions.end();
           ++i)
       {
+      if(i->compare("baa") != 0) //FIXME This conditoin can be removed when
+      // the output of `echo Aaa|nuspell -d en_US` no longer contains "baa"
+      // as a suggestion. This change in Nuspell is planned but not yet
+      // available. The Nuspell team has also an internal issue reminding them

```

Comment:
If the Nuspell issue is on GitHub, could you provide the URL, please?

Reply:
Soory, no it is not, it is in a private issue tracker, only available for the project members.